### PR TITLE
Implement string class extensions. 

### PR DESCRIPTION
**Pull request by azemoh**

### CODE REVIEW

File: src/string.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+/**
+ * Test if a string is a question
+ * @return {Boolean}
+ */
+String.prototype.isQuestion = function () {
+  return /\?$/.test(this);
+};
+
+/**
+ * Return a array of words in a string
```
Comment (andela-eakinyele) [inline src/string.js]: 😄  Returns an array of ...

File: src/string.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    // remove last "," 654,321, => 654,321
+    .replace(/,$/, '')
+    // reverse back 654,321 => 123,456
+    .reverse();
+
+  return `${str[0]}.${str[1].substr(0, 2)}`;
+};
+
+/**
+ * Get number from a string
```
Comment (andela-eakinyele) [inline src/string.js]: parse a string to a number

File: src/string.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  let mid = this.length / 2;
+  return (mid === parseInt(mid)) ? this.substr(mid - 1, 2) : this.charAt(mid);
+};
+
+/**
+ * Convert numbers in a string to text
+ * @return {String}
+ */
+String.prototype.numberWords = function () {
+  const table = {
```
Comment (andela-eakinyele) [inline src/string.js]: do you mind renaming this to numStringMap

File: src/string.js [hunk 4]
```diff
@@ -0,0 +1,161 @@
+/**
+ * Test if string containes a vowel
+ * @return {Boolean}
+ */
+String.prototype.hasVowels = function () {
```
Comment (andela-celisha-wigwe) [inline src/string.js]: Neat code. @azemoh. About your use of `functions`, why not stick to es6? Like this:
```
String.prototype.hasVowels = () => {
  return /[aeiou]/i.test(this);
}
```
You can use `bind(this)` where it is required.

**Pull request merged**